<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordSpelling</class>
 <widget class="QDialog" name="WordSpelling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="top_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>66</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>271</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="meaning_label">
      <property name="text">
       <string>释义：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="meaning_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="new_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>109</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>生词</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>107</x>
     <y>131</y>
     <width>159</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="wordfirst_label">
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="word_lineEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="wordlast_label">
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
